Tasks/app-store-release/app-store-release.ts

```diff
@@ -43,6 +43,27 @@ function findIpa(ipaPath: string): string {
     return paths[0];
 }
 
+async function isFastlaneInstalled(): Promise<boolean> {
+    tl.debug('Checking for previous fastlane version first...');
+    let gemRunner: ToolRunner = tl.tool(tl.which('gem', true));
+    gemRunner.arg(['list', 'fastlane']);
+    gemRunner.arg(['-i']);
```

Comment:
It's not clear to me why this is on a different line than the ones above. e.g. `['list', 'fastlane', '-i']`

Reply:
I originally proposed that code change to align with the same format of the lines 113 and 114
` gemRunner.arg(['uninstall', 'fastlane']);`
` tl.debug(`Uninstalling all fastlane versions...`);`
`gemRunner.arg(['-a', '-I']);  //uninstall all versions`

Reply:
Thanks!  It doesn't really matter as the code is behaviorally the same; it wasn't clear to me how the grouping improves readability.

Reply:
That's right, either way is fine. Since there is only one argument on the 2nd line, I will merge them into one line.